src/main/resources/schema/schema_v28.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE system_columns
+  ADD COLUMN separator BOOLEAN;
```

Comment:
Wenn der default im code `false` ist, sollte das auch im alter statement so sein, damit die Daten korrekt migriert werden und wir nicht `NULL`s handeln müssen, auch wenn das zu `false` mapped.
`ADD COLUMN separator BOOLEAN DEFAULT 'FALSE' NOT NULL;` sollte es tun.